providers/summary/scancode.js

```diff
@@ -153,9 +154,16 @@ class ScanCodeSummarizer {
     return result.length === 0 ? null : result
   }
 
-  _collectAttributions(copyrights, holders) {
+  _collectAttributions(copyrights) {
     if (!copyrights || !copyrights.length) return null
-    return this._setToArray(copyrights.reduce((result, entry) => this._addArrayToSet(entry.holders, result), new Set()))
+    const set = new Set()
+    for (let copyright of copyrights) {
+      if (!copyright.statements) continue
+      const statements = copyright.statements.map(statement => he.decode(statement)
+        .replace(/\\[nr]/, '\n'))
+      this._addArrayToSet(statements, set)
+    }
+    return this._setToArray(set)
```

Comment:
Does the call need to change as we are now returning the array rather than modifying the `holders` array that was passed in?

Reply:
I think it should still be the same (we still want to build a deduped list of statements and return an array)